### ISSUE TITLE
MD025 and MD023 false positive in code blocks after single line code block

### ISSUE THREAD

**Issue by nikolaik** [rvben/rumdl#175]
MD025 and MD023 checks should not trigger inside code blocks and doesn't in the normal case. Though if I create a document containing a single line code block (is that even valid?) the parser gets confused after that.

```
```something```
```

followed by a this code block:

```
```bash
# Have a parrot
echo "🦜"
```
```

**Comment by nikolaik** [comment]
And love the project ❤️ So fast! well done 👏👏👏

**Comment by rvben** [comment]
Hi @nikolaik, thanks for the issue and the kind words. The parser indeed got confused while it shouldn't have been. This is fixed in 0.0.187. 🚀
